src/devtools/client/webconsole/components/Input/JSTerm.js

```diff
@@ -52,30 +52,30 @@ class JSTerm extends Component {
     return {
       // Returns previous or next value from the history
       // (depending on direction argument).
-      getValueFromHistory: PropTypes.func.isRequired,
```

Comment:
Guessing we're just simplifying a lot here? In which case we can probably get rid of all the comments.